riscv/src/continuations.rs

```diff
@@ -45,6 +47,11 @@ fn transposed_trace<F: FieldElement>(trace: &ExecutionTrace<F>) -> HashMap<Strin
 }
 
 fn render_memory_hash<F: FieldElement>(hash: &[F]) -> String {
+    // TODO: support for small field, which uses two F per word
```

Comment:
here or later?

Reply:
is this already written in the other branch? or should we make an issue so we don't forget?

Reply:
the assertion is enough: you'll get a panic if you try to run with small field